include/libdnf/repo/repo.hpp

```diff
@@ -391,7 +391,11 @@ class Repo {
 
     void internalize();
 
-    void reset_metadata_expired();
+    /// If the repository is not already marked as expired, it checks for the presence of the storage cache expiration
+    /// attribute, and if the metadata_expire configuration value is set, also checks the modification times
+    /// of the master configuration file, the repository configuration file, and the cached primary file.
+    /// Depending on the result, the repository may be marked as expired.
+    void recompute_expired();
```

Comment:
[note - minor] In theory `recompute_expired()` can return recomputed value of expire and it will spare one line in code.

Reply:
[minor] What about to mention that we have 3 methods playing with expire:
bool is_expired() const;
void expire();
void recompute_expired()